src/screens/Vault/HardwareModalMap.tsx

```diff
@@ -1016,12 +1010,12 @@ function HardwareModalMap({
     const findSigningServer = async (otp) => {
       try {
         setInProgress(true);
-        if (unkownSigners.length <= 1)
+        if (vaultSigners.length <= 1)
           throw new Error('Add two other devices first to do a health check');
-        //confirm if mfp
-        // if fingerPrint then derive masterFingerprints
-        // f() to derive mfp ?
-        const ids = unkownSigners.map((signer) => signer.masterFingerprint);
+        const network = WalletUtilities.getNetworkByType(config.NETWORK_TYPE);
+        const ids = vaultSigners.map((signer) =>
+          WalletUtilities.getFingerprintFromExtendedKey(signer.xpub, network)
+        );
```

Comment:
VaultSigner already has xfp in it? Do we need to rederive the fingerprint using its xpub?